mediaqueries/Overview.bs

```diff
@@ -1709,6 +1713,10 @@ Hover Capability: the 'hover' feature</h3>
 	For accessibility reasons, even on devices that do support hovering,
 	the UA may give a value of ''hover: none'' to this media query,
 	to opt into layouts that work well without hovering.
+	In addition, even if the primary input mechanism has 'hover: hover' capability,
+	there may be additional input mechanisms available to the user where 'hover: none' is true.
+	Authors may	wish to query the 'any-hover' media feature to take these other non-hover-capable
+	potential input mechanisms into account.
```

Comment:
This does not work. While there may be other devices which cannot hover (and arguably this is a common case: every desktop computer with a mouse also has a keyboard that cannot hover), the query will not let you detect that in the way you suggest. `any-hover: none` only matches if none of the devices can hover, not merely if some of them cannot.

Maybe we should change that (although I am not convinced), but until we do, this note is incorrect.